web/server.go

```diff
@@ -613,6 +613,12 @@ func (s *Server) servePrintErr(w http.ResponseWriter, r *http.Request) error {
 
 	f := result.Files[0]
 
+	if qvals.Get("format") == "raw" {
+		w.Header().Add("Content-Type", "text/plain")
```

Comment:
I think this is a bit safer from a security perspective. The last time I implemented a raw endpoint we used these headers.

```suggestion
		w.Header().Set("Content-Type", "text/plain; charset=utf-8")
		w.Header().Set("X-Content-Type-Options", "nosniff")
```